src/api/java/mezz/jei/api/gui/handlers/IGuiClickableArea.java

```diff
@@ -15,9 +15,21 @@
 	 * The hover/click area for this {@link IGuiClickableArea}.
 	 * When hovered, the message from {@link #getTooltipStrings()} will be displayed.
 	 * When clicked, {@link #onClick(IFocusFactory, IRecipesGui)} will be called.
+	 * @deprecated Prefer and implement {@link #isMouseOver(double, double)} instead.
 	 */
+	@Deprecated//TODO 1.16: Remove this and make isMouseOver no longer defaulted
```

Comment:
I don't think this needs to be deprecated, it is simpler to implement for the vast majority of mods, and only a slight inconvenience for anyone needing `isMouseOver`.